packages/create-sitecore-jss/src/templates/react/src/lib/dictionary-service-factory.js

```diff
@@ -0,0 +1,14 @@
+import { RestDictionaryService } from '@sitecore-jss/sitecore-jss-react';
```

Comment:
As I understand, we don't have `jss-create` file anymore, and in this case we don't have file replacement. So, you should have a single factory which will use ejs templating, to use Graphql or Rest service

Reply:
By this you mean we should have an environment variable to check if we are fetching with GraphQL or REST?

Reply:
Only next supports .env out of the box, the other frameworks will use ejs conditionals, for example: https://github.com/Sitecore/jss/blob/diy-generator-angular/packages/create-sitecore-jss/src/templates/angular/src/app/lib/layout-service-factory.ts

Reply:
Thanks, @CobyPear. I was searching for this.